src/containers/NostrLive.js

```diff
@@ -0,0 +1,191 @@
+/* eslint-disable react/no-array-index-key */
```

Comment:
Would it be better to just replace the OnSaleOrdinalsArea.js file?

Reply:
I don't think OnSaleOrdinalsArea it's an appropiate name for that component tbh